syntax = "proto3";

package ecommerce;

service Seller {
  rpc RegisterSeller (SellerRequest) returns (RegisterResponse);
}

message SellerRequest {
  string seller_id = 1;
}

message RegisterResponse {
  string status = 1;
}
